dockerfiles/init/modules/che-kubernetes-helm/readme.md

```diff
@@ -32,55 +32,53 @@ The context of the commands below is the directory in which this readme file res
+  helm upgrade --install <my-che-installation> --namespace <my-che-namespace> --set global.ingressDomain=<my-hostname> --set cheImage=<my-image> ./
   ```
 
-#### Deployment types
-Currenty, only minikube deployment is supported.
+#### Deployment Options
 
 ##### Single User 
 Only Che will be deployed.
 
   ```bash
-  helm upgrade --install <che-release> --namespace <che-namespace> --set global.cheDomain=<domain> ./
+  helm upgrade --install <che-release-name> --namespace <che-namespace> --set global.ingressDomain=<domain> ./
   ```
   
 ##### Multi User 
 Che, KeyCloak and Postgres will be deployed.
 
   ```bash
-  helm upgrade --install <che-release> --namespace <che-namespace> --set global.multiuser=true --set global.cheDomain=<domain> ./
+  helm upgrade --install <che-release-name> --namespace <che-namespace> -f ./values/multi-user.yaml --set global.ingressDomain=<domain> ./
   ```
-  
-##### No Host:
- Ingress will serve requests on minikube-ip. 
- Path based routing to Che, Secondary servers (KeyCloak) and Workspace servers.
 
+#### Default Host
+All Ingress specs are created without a host attribute (defaults to *).
+Path based routing to all components.
+Multi User configuration is enabled. 
+ 
   ```bash
-  helm upgrade --install <che-release> --namespace <che-namespace> --set global.isHostbased=false --set global.cheDomain=<minikube-ip> ./
-   Master: http://<minikube-ip>/
-   Workspaces: http://<minikube-ip>/<path-to-server>
-   Keycloak (if multiuser) : http://<minikube-ip>/auth/
+  helm upgrade --install <che-release-name> --namespace <che-namespace> -f ./values/default-host.yaml --set global.ingressDomain=<domain> ./
   ```
-  
-##### Host (partial): 
- WS Master Ingress will serve requests on provided domain
- Workspaces: Ingress will serve requests on minikube-ip, Path Based routing to workspaces. 
- KeyCloak : dedicated hostname 
-   
-   ```bash
-   helm upgrade --install <che-release> --namespace <che-namespace> --set global.cheDomain=<minikube-ip>.xip.io ./
-   Master: http://master.<minikube-ip>.xip.io
-   Workspaces: http://<minikube-ip>/<path-to-server>
-   Keycloak (if multiuser): http://keycloak.<minikube-ip>.xip.io/
-   ```
  
+* Master: `http://<minikube-ip>/`
+* Keycloak:  `http://<minikube-ip>/auth/`
+* Workspaces servers: `http://<minikube-ip>/<path-to-server>`
+
+#### TLS-enabled
+Cert-Manager is used to issue LetsEncrypt certificates.
+To avoid rate-limit issues, we use a single hostname for all ingresses.
+Path based routing to all components.
+Multi User configuration is enabled. 
+
+  ```bash
+  helm install --name <cert-manager-release-name> stable/cert-manager
+  helm upgrade --install <che-release-name> --namespace <che-namespace> -f ./values/tls.yaml --set global.ingressDomain=<your-domain> ./
+  ```
+
+* Master: `https://che-<che-namespace>.your-domain/`
+* Keycloak:  `https://che-<che-namespace>.your-domain/auth/`
+* Workspaces servers: `https://<che-namespace>.your-domain/<path-to-server>`
```

Comment:
Workspaces servers should be in the format https://**che-**<che-namespace> too (with the `che-` prefix) or am I missing something?

Reply:
done